Velaptor/Velaptor.csproj

```diff
@@ -45,6 +45,11 @@
     <OutputPath>bin\x64\Release\</OutputPath>
   </PropertyGroup>
 
+<ItemGroup>
+  <InternalsVisibleTo Include="$(AssemblyName)Tests" />
+  <InternalsVisibleTo Include="DynamicProxyGenAssembly2" />
+</ItemGroup>
```

Comment:
Indent to align with the rest

![image](https://user-images.githubusercontent.com/105235056/209815983-0a5c6978-46e0-4b1b-91e9-d5d623a736e2.png)